app/script/calling/entities/Call.coffee

```diff
@@ -392,7 +388,7 @@ class z.calling.entities.Call
 
   ###
   Get full flow telemetry report of the call.
-  @return [Array<z.calling.Flow>] Array of flows
+  @return [Array<Object>] Array of flow telemetry reports for calling service automation
```

Comment:
Good idea, but will skip for now. Only using this is the calling service automation.